easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.3.2-foss-2023a-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,163 @@
+easyblock = 'PythonBundle'
+
+name = 'AlphaFold'
+version = '2.3.2'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://deepmind.com/research/case-studies/alphafold'
+description = "AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known"
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+builddependencies = [
+    ('poetry', '1.5.1')
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('CUDA', '12.1.1', '', SYSTEM),
+    ('SciPy-bundle', '2023.07'),
+    ('PyYAML', '6.0'),
+    ('TensorFlow', '2.13.0'),  # doesn't require TF-gpu
```

Comment:
2.15.1 might be better?

Reply:
CPU-only TensorFlow is 2.13.0 in `2023a` generation, so this is fine (people can always swap to 2.15.1, but I doubt it really matters for AlphaFold)